source/releasenotes/2024-01-24-wp-cli-v2-9-0-release-now-available-on-pantheon.md

```diff
@@ -0,0 +1,21 @@
+---
+title: WP-CLI v2.9.0 release now available on Pantheon
+published_date: "2024-01-25"
```

Comment:
@IngridKwok can you update this so it matches filename? 

`published_date: "2024-01-24"`